validation/messaging-system/cfr/cfr.1.prerelease.pass.json

```diff
@@ -0,0 +1,13 @@
+{
+  "client_id": "94642acb-4996-034b-916c-147da723cc41",
+  "impression_id": "n/a",
```

Comment:
I think these new test cases are being skipped due to having an extra level of directories. The expected pathing is different here from `templates` and there should be no intermediate `cfr` directory.

I think these test will fail because we have regexes on impression_id and client_id that require UUIDs and should fail on "n/a".

The value of "n/a" may cause problems in the pipeline, too, since we specially look for `client_id` if it exists in the payload in order to calculate sample_id. We may make an assumption that if it's present it's a valid UUID.

Reply:
Nice catch. Since `impression_id` and `client_id` form a `oneOf` relation here, we should just omit the other one if one is present in the payload.